tests/decryption/test_decrypt.py

```diff
@@ -11,7 +11,8 @@
     get_private_keys,
     decrypt_files,
     move_files,
-    remove_files
+    remove_files,
+    get_args
```

Comment:
When you "list" params/args like that, please always add a trailing comma to the last item as well. It makes the diff view cleaner when changed (as demonstrated here).

Also, please order in ascencing alphanumeric order.

Finally, it seems that you are adding a test for `get_args()`, not `remove_file()`, no? If so, please adjust the PR title accordingly.

Reply:
This PR is for `remove_file()` but I merged the `get_args()` tests into main first that's my bad.